deploy.sh

```diff
@@ -23,7 +23,7 @@ fi
 
 msg "Removing the old website"
 pushd public
-git rm -rf *
+git rm -rf public/*
```

Comment:
Sorry, just a final thought. In `deploy.sh` the 3 lines could be merged into `git rm -rf public/*` which only deletes everything inside `public/`, and the folder would be preserved. Would you agree to that?